docs/06-concepts/11-authentication/04-providers/06-custom-providers.md

```diff
@@ -165,3 +248,35 @@ if (serverResponse.success) {
 ```
 
 The example above shows how to register a signed-in user in the session manager.
+
+### Sign Out
+
+The `SessionManager` provides multiple ways to handle sign-out functionality, allowing you to either sign out from the current device or all devices:
```

Comment:
Chore: We describe the session manager under `Setup` as well. Make sure that description matches our new behavior.